src/screens/AppIntroScreen.tsx

```diff
@@ -68,7 +68,7 @@ const SliderButton = ({
   const buttonStyle = isDisabled ? [style, styles.sliderButtonDisabled] : style;
 
   return (
-    <TouchableOpacity onPress={onPress} style={buttonStyle} disabled={isDisabled}>
+    <TouchableOpacity onPress={onPress} style={buttonStyle}>
```

Comment:
how is the flow now with not disabling the button? it looks like disabled but is still pressable to move on?